packages/core/src/global/js/accessibility.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-inner-declarations */
+
+function setupAccessibility() {
+  if (typeof document !== 'undefined') {
```

Comment:
Should this be `window.document`?

Reply:
It's not necessary.

Reply:
`typeof` will throw an error because something is undefined. It will just return `undefined`

Reply:
ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#Errors

Reply:
Awesome!  Totally forgot about this safety with typeof.